INFO     test_execution:conftest.py:240 Starting test: tests/ui/test_inventory_list_count.py::test_inventory_displays_exactly_six_items[webkit] (setup)
INFO     browser.console:conftest.py:121 [log] Content is cached for offline use.
INFO     browser.console:conftest.py:121 [error] Failed to load resource: the server responded with a status of 401 (Unauthorized)
INFO     browser.console:conftest.py:121 [error] Failed to load resource: the server responded with a status of 401 (Unauthorized)
INFO     test_execution:conftest.py:251 Test completed: tests/ui/test_inventory_list_count.py::test_inventory_displays_exactly_six_items[webkit] (setup) (duration: 2.58s)